dsl/pace/dsl/dace/dace_config.py

```diff
@@ -77,6 +77,10 @@ def __init__(
                 "args",
                 value="-std=c++14 -Xcompiler -fPIC -O3 -Xcompiler -march=native",
             )
+            dace.config.Config.set("compiler", "cuda", "cuda_arch", value="60")
+            dace.config.Config.set(
+                "compiler", "cuda", "default_block_size", value="64,8,1"
+            )
```

Comment:
Ignore this if it's obvious, but is this valid to do in all cases? What if we run on small domain sizes?

Reply:
We would under-perform  for small domain size, indeed.
A real code for this would actually read the hardware capacities & the experiment size and come up with a setting that matches a bit better, since arguably 64,8 for a H100 is pretty weak (even for A100).
Logged in JIRA

Reply:
Sorry for the noise, but could you put that as an issue on GitHub, and add a comment above with a link to the issue? At some point we'll lose the JIRA access and it's a code issue directly in Pace, so it's probably easier to find on GitHub. Thanks!

Reply:
Good point, should be doing this all the time!